docs/config.md

```diff
@@ -0,0 +1,89 @@
+# Configuration
+
+These are the config settings available to jj/Jujutsu.
+
+The config settings are located at `~/.jjconfig.toml`.
+
+See the [TOML site](https://toml.io/en/) for more on syntax,
+but one thing to remember is that anything under a heading can be dotted
+e.g. `user.name = "YOUR NAME"` is equivalent to:
+
+    [user]
+      name = "YOUR NAME"
+
+Headings only need to be set once in the real config file but the examples below repeat the headings so it's clear where they go.
+
+The other thing to remember is that the value of a setting (the part to the right of the `=` sign) should be surrounded in quotes if it's a string. That's probably enough TOML to keep you out of trouble but the syntax guide is very short if you ever need to check.
+
+
+## User settings
+
+    [user]
+      name = "YOUR NAME" 
+      email = "YOUR_EMAIL@example.com"
+
+Don't forget to change these to your own details!
+
+
+## UI settings
+
+These all go under the `[ui]` heading.
+
+
+### Colorizing output
+
+Possible values are `always`, `never` and `auto` (default: `auto`). 
+`auto` will use color only when writing to a terminal. 
+
+This setting overrides the `NO_COLOR` environment variable (if set).
+
+    [ui]
+      color = "never" # Turn off color
```

Comment:
I wonder if we'll need examples like this for every variable. The examples are always needed with the current structure of the document because that's the only place the key name is defined. I noticed that `git help config` and `hg help config` instead use the key name as the section heading. This seems fine for now, however; we can consider restructuring it later when this doc gets longer.